constantine/src/types/g1.rs

```diff
@@ -450,11 +450,50 @@ impl G1Affine<CtG1, CtFp> for CtG1Affine {
     }
 
     fn to_bytes_uncompressed(&self) -> [u8; 96] {
-        todo!()
+        let mut out = [0u8; 96];
+        // Serialize: 48 bytes x (big-endian) || 48 bytes y (big-endian)
+        // limbs are stored in little-endian, so limbs[5] is most significant
+        for i in 0..6 {
+            let bytes = self.0.x.limbs[5 - i].to_be_bytes();
+            out[i * 8..(i + 1) * 8].copy_from_slice(&bytes);
+        }
+        for i in 0..6 {
+            let bytes = self.0.y.limbs[5 - i].to_be_bytes();
+            out[48 + i * 8..48 + (i + 1) * 8].copy_from_slice(&bytes);
+        }
+        out
```

Comment:
That's actually not correct serialization format, it doesn't handle infinity points. Here is [blst backend notes on serialization](https://github.com/supranational/blst?tab=readme-ov-file#serialization-format). You can find somewhere online zcash serialization format, or just port blst serialization implementation, as internal representation of both blst and constantine backends are the same. https://github.com/supranational/blst/blob/806515251de5cfb9761ad12eb2f2f151b1ad0277/src/e1.c#L177